src/components/Shared/Toast/ToastStyles.css

```
@@ -0,0 +1,906 @@
+}
+
+@keyframes Toastify__bounceInRight {
+  from,
+  60%,
+  75%,
+  90%,
+  to {
+    animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+  }
+
+  from {
```

Biome (1.9.4)

[error] The duplicate keyframe selector is overwritten by later one.

Consider using a different percentage value or keyword to avoid duplication

(lint/suspicious/noDuplicateSelectorsKeyframeBlock)